pyannote/audio/applications/change_detection.py

```diff
@@ -0,0 +1,411 @@
+    pyannote-change-detection -h | --help
+    pyannote-change-detection --version
+
+Options:
+    <experiment_dir>           Set experiment root directory. This script expects
+                                                         a configuration file called "config.yml" to live
+                                                         in this directory. See "Configuration file"
+                                                         section below for more details.
+    <database.task.protocol>   Set evaluation protocol (e.g. "Etape.SpeakerDiarization.TV")
+    <wav_template>             Set path to actual wave files. This path is
+                                                         expected to contain a {uri} placeholder that will
+                                                         be replaced automatically by the actual unique
+                                                         resource identifier (e.g. '/Etape/{uri}.wav').
+    <train_dir>                Set path to the directory containing pre-trained
+                                                         models (i.e. the output of "train" mode).
+    <tune_dir>                 Set path to the directory containing optimal
+                                                         hyper-parameters (i.e. the output of "tune" mode).
+    --database=<db.yml>        Path to database configuration file.
+                               [default: ~/.pyannote/db.yml]
+    --subset=<subset>          Set subset (train|developement|test).
+                               In "train" mode, default is "train".
+                               In "validation" mode, default is "development".
+                               In "tune" mode, default is "development".
+                               In "apply" mode, default is "test".
+    -h --help                  Show this screen.
+    --version                  Show version.
+
+Configuration file:
+        The configuration of each experiment is described in a file called
+        <experiment_dir>/config.yml, that describes the architecture of the neural
+        network used for sequence labeling (0 vs. 1, non-speech vs. speech), the
+        feature extraction process (e.g. MFCCs) and the sequence generator used for
+        both training and testing.
+
+        ................... <experiment_dir>/config.yml ...................
+        feature_extraction:
+             name: YaafeMFCC
+             params:
+                    e: False                   # this experiments relies
+                    De: True                   # on 11 MFCC coefficients
+                    DDe: True                  # with 1st and 2nd derivatives
+                    D: True                    # without energy, but with
+                    DD: True                   # energy derivatives
+                    stack: 3
```

Comment:
In this example, can you make sure you are using the same parameters as in the paper?
In particular, I think it should read `stack: 1`, and maybe set `duration` and `step` to their correct value?